server/core/lu_packet.py

```diff
@@ -1,22 +1,37 @@
+"""
+Packet router
+"""
+
 import logging
-from pyraknet.bitstream import c_uint8, c_uint16, c_uint32, ReadStream
+
+from pyraknet.bitstream import ReadStream
 from plugin import Plugin, Action, Packet
 from structs import LUHeader
 
+
 log = logging.getLogger(__name__)
 
+
 class PacketRouter(Plugin):
+    """
+    Packet router plugin
+    """
     def actions(self):
+        """
+        Returns all actions
+        """
         return [
             Action('rn:user_packet', self.on_packet, 10)
         ]
-    
+
     def on_packet(self, data, address):
+        """
+        Packet handler
+        """
         packet = Packet.deserialize(ReadStream(data), self.server.packets)
-        
-        if getattr(packet.header,'packet_name', None) is None:
+
+        if not getattr(packet.header, 'packet_name', None):
```

Comment:
Do we want to get rid of the explicit final `None` in calls to `getattr`? At some point I thought that was required, but thats note the case